cfme/tests/automate/custom_button/test_infra_objects.py

```diff
@@ -465,11 +488,18 @@ def _reset_window():
             1668023,
             forced_streams=["5.10"],
             unblock=lambda button_group, btn_dialog: not ("HOSTS" in button_group and btn_dialog),
-        )
+        ),
+        BZ(
+            1685555,
+            unblock=lambda button_group, btn_dialog: not ("SWITCH" in button_group and btn_dialog),
+        ),
     ]
 )
 @pytest.mark.ignore_stream("5.9")
 @pytest.mark.parametrize("btn_dialog", [False, True], ids=["simple", "dialog"])
+@pytest.mark.uncollectif(
+    lambda appliance, button_group: appliance.version < "5.10" and "SWITCH" in button_group
```

Comment:
this `uncollect` is unnecessary as you have `ignore_stream("5.9")`

Reply:
@lina-nikiforova this comment must in `Required` category. In feature please put `Required` tag and move PR to `WIP`